lib/Backend.php

```
@@ -0,0 +1,227 @@
+
+        return
+            '<div class="rex-minibar-info-group">
+                <div class="rex-minibar-info-group-title">' . rex_i18n::msg('metainfo_minibar_clang_title') . '</div>
+                ' . implode('', $items) . '
+            </div>';
+    }
+
+    /** @param rex_extension_point<array<string,array<string>>> $ep */
+    public static function epPageBodyAttr(rex_extension_point $ep): void
+    {
+        if (false !== Minibar::getInstance()->isActive()) {
```

PHPStan (2.1.54)

[error] Call to an undefined static method FriendsOfRedaxo\Minibar\Minibar::getInstance().


(staticMethod.notFound)